http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -257,14 +257,30 @@ object Boot extends IOApp {
               appDependencies.asyncTasksQueue
             )
 
+          implicit val clusterToolToToolDao =
```

Comment:
Yeah the naming is maybe not ideal. :( `LeoMetricsMonitor` deals with both runtimes and apps and reports both metrics. `clusterToolToToolDao` and `ToolDAO` deals just with runtimes. The `WdsDAO`, `CromwellDAO`, etc are apps. Wonder if this could be made clearer with comments or something..